resources/js/Pages/Home.vue

```diff
@@ -162,20 +181,22 @@
             validate(): void {
                 this.validationError = null;
 
+                const typeError = 'error';
+
                 const rules = [{
                     check: (ids: Array<string>) => ids.length < 1,
-                    type: 'warning',
-                    message: this.$i18n('item-form-error-message-empty')
+                    type: typeError,
```

Comment:
I added a variable because at some point there will be a warning and because it needs to be the same name in the type and they key of the message. So they are not changed separately.